### ISSUE TITLE
[UX] restore messages are confusing

### ISSUE THREAD

**Issue by arthurlutz** [YunoHost/issues#1238]
When trying to restore after a failed upgrade I try out the restore tool
```
root@yunohost:~# yunohost backup restore peertube-pre-upgrade1 --debug
316  DEBUG loading actions map namespace 'yunohost'
372  DEBUG extra parameter classes loaded: ['ask', 'password', 'required', 'pattern']
373  DEBUG initializing base actions map parser for cli
376  DEBUG registering new callback action 'yunohost.utils.packages.ynh_packages_version' to ['-v', '--version']
620  DEBUG initialize authenticator 'ldap-anonymous' with: uri='ldap://localhost:389', base_dn='dc=yunohost,dc=org', user_rdn='None'
633  DEBUG stale lock file found
633  DEBUG lock has been acquired
788  DEBUG loading python module yunohost.backup took 0.154s
788  INFO processing action [3302.1]: yunohost.backup.restore with args={'force': False, 'name': 'peertube-pre-upgrade1', 'apps': None, 'system': None, 'auth': <moulinette.authenticators.ldap.Authenticator object at 0x7fabda4c1090>}
789  WARNING YunoHost is already installed
Do you really want to restore an already installed system? [y/N]: y
```
Since the app I am trying to restore is not installed anymore the question is confusing. Before hitting "yes" I though this might re-install YunoHost (probably because of the warning above). 

Come to re-read the log above, how come yunohost thinks the application is still installed ?

**Comment by alexAubin** [comment]
Hellow,

are you running Yunohost 2.x or 3.x ?

**Comment by alexAubin** [comment]
Ah now that I think about it, I guess you are running YunoHost 3.x. I was suspecting this would not happen anymore on 3.x, but yes it will...

Reason behind this is that if you do not declare what you wish to restore from this archive, it will restore everything (apps and system parts) ... So because it will restore system parts, it asks if "you really want to restore an already installed system?".

But that's not very logical, because your archive probably only contain an app backup... So the fix would be to check if we really are going to restore any system parts before asking this.

**Comment by arthurlutz** [comment]
Indeed, the restore command ends up on 

```
apps: 
  peertube: Success
system: 
```

Which I guess says "nothing was restored on the system side". +1 to add this check and not ask the question if not needed.

**Comment by alexAubin** [comment]
Got fed up of seeing this once again, should be fixed in 4.1 : https://github.com/YunoHost/yunohost/commit/0aaaeef807556540c29a0c7d6d9a1d8025b423d6
